src/main/plugins/simple-folder-search-plugin/simple-folder-search-plugin.ts

```diff
@@ -72,6 +73,11 @@ export class SimpleFolderSearchPlugin implements SearchPlugin, OpenLocationPlugi
             resolve();
         });
     }
+    public autoComplete(searchResultItem: SearchResultItem): string {
+        return searchResultItem.executionArgument.endsWith(sep)
+            ? searchResultItem.executionArgument
+            : `${searchResultItem.executionArgument}${sep}`;
```

Comment:
What if the `searchResultItem` is a file? I guess then the `sep` is not needed.

Reply:
Okay but That's is the same method from file-browser-plugin. Should I change that too or leave it ?